backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -706,36 +702,59 @@ public void updateUserPrivilegesAndGroupAccess(
       String orgExternalId,
       boolean allFacilitiesAccess,
       List<UUID> facilities,
-      OrganizationRole role)
+      Role role)
       throws IllegalGraphqlArgumentException {
 
-    if (!allFacilitiesAccess && facilities.isEmpty()) {
```

Comment:
the diff here is a little misleading but I essentially moved the validation logic out of `updateUserPrivilegesAndGroupAccess` into `getFacilitiesToGiveAccess` so I could call it from multiple places